### PR TITLE
AddinUtil LOLBAS Detections

### PR DESCRIPTION
**Pull request by SILJAEUROPA**
### Summary of the Pull Request

4 new detections aimed at detecting malicious usage of the Microsoft .NET Binary AddinUtil.exe (ref: https://www.blue-prints.blog/content/blog/posts/lolbin/addinutil-lolbas.html).

### Changelog

new:	Network Connection Initiated By AddinUtil.EXE
new: Suspicious AddinUtil.EXE CommandLine Execution
new: Uncommon Child Process Of AddinUtil.EXE
new: Uncommon AddinUtil.EXE CommandLine Execution
new: AddinUtil.EXE Execution From Uncommon Directory

### Example Log Event

N/A

### Fixed Issues

N/A

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

**Review by nasbench — APPROVED** [review]

**Comment by nasbench** [comment]
Thanks for the awesome contribution @SILJAEUROPA and GGs for the find.

I updated some metadata to align with the Sigma repo and added another variant of the rule with a higher level to account for potential suspicious paths that might be abused.

Cheers

**Comment by SILJAEUROPA** [comment]
Cheers nas, thanks for additions and kind words

File: rules/windows/network_connection/net_connection_win_addinutil.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+title: Network Connection Initiated By AddinUtil.EXE
+id: 5205613d-2a63-4412-a895-3a2458b587b3
+status: experimental
+description: Detects network connections made by the Add-In deployment cache updating utility (AddInutil.exe), which could indicate command and control.
```
Comment (phantinuss) [inline rules/windows/network_connection/net_connection_win_addinutil.yml:4]:
```suggestion
description: Detects network connections made by the Add-In deployment cache updating utility (AddInutil.exe), which could indicate command and control communication.
```

File: rules/windows/process_creation/proc_creation_win_addinutil_uncommon_child_process.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+title: Uncommon Child Process Of AddinUtil.EXE
+id: b5746143-59d6-4603-8d06-acbd60e166ee
+status: experimental
+description: |
+    Detects uncommon child process of the Add-In deployment cache updating utility (AddInutil.exe) which could be a sign of potential abuse of the binary to proxy execution via a custom Addins.Store payload.
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_addinutil_uncommon_child_process.yml:5]:
```suggestion
    Detects uncommon child processes of the Add-In deployment cache updating utility (AddInutil.exe) which could be a sign of potential abuse of the binary to proxy execution via a custom Addins.Store payload.
```

**Review by phantinuss — APPROVED** [review]

**Pull request merged**